### ISSUE TITLE
Tempo datasource doesn't show errors when failing to fetch traces

### ISSUE THREAD

**Issue by tomwilkie** [grafana/grafana#32990]
**What would you like to be added**:

I'm getting 500s fetching my trace from the grafana backend:

![image](https://user-images.githubusercontent.com/444037/114713878-75c50c00-9d29-11eb-9115-d5f17ed30d52.png)

In the grafana logs, this is due to some network issue or misconfiguration:

```
t=2021-04-14T12:57:14+0000 lvl=eror msg="Metric request error" logger=context userId=0 orgId=1 uname= error="failed get to tempo: Get \"http://tempo:3100//api/traces/12a81b698859e90b\": dial tcp 172.22.0.4:3100: connect: connection refused" remote_addr=172.22.0.1
```

Would be good to show this in UI.

**Why is this needed**:

Makes debugging bad setups easier?

**Comment by tomwilkie** [comment]
Grafana v7.5.3

**Comment by zoltanbedi** [comment]
This is fixed in 8.0.0
